guides/keras_tuner/getting_started.py

```diff
@@ -69,7 +69,7 @@ def build_model(hp):
 
 """
 There are many other types of hyperparameters as well. We can define multiple
-hyperparameters in the function. In the following code, we tune the whether to
```

Comment:
This is an actual typo

Reply:
Thank you for your response. I have made the changes as required.